app/helpers/application_helper.rb

```diff
@@ -1,2 +1,19 @@
 module ApplicationHelper
+  def distance
+    # position 0 is the displayed text and position 1 is the tag
+    [
+      ['1 Mile', '1 Mile'],
```

Comment:
These numbers will conflict with current branch that has been pulled down and will need to be dealt with before a merge.